src/taskFunctions.js

```diff
@@ -0,0 +1,29 @@
+export const saveTasks = (tasks) => {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+export const addTask = (tasks, description) => {
+  const task = {
+    description,
+    completed: false,
+    index: tasks.length + 1,
+  };
+  tasks.push(task);
+  saveTasks(tasks);
+  return tasks;
+};
+
+export const deleteTask = (tasks, index) => {
+  tasks.splice(index, 1);
+  tasks.forEach((task, i) => {
+    task.index = i + 1;
+  });
+  saveTasks(tasks);
+  return tasks;
+};
```

Comment:
- [ ] as part of the requirements Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. , kindly fix that since currently the tasks id are not updated after deleting 🙏